src/index.test.ts

```diff
@@ -92,3 +94,25 @@ test('export', (t): void => {
 
   t.deepEqual(exported, expected)
 })
+
+test('Own peerDependencies include those of eslint-config-standard', (t) => {
+  Object
+    .entries(standardPkg.peerDependencies)
+    .forEach(([_name, standardRange]) => {
+      // https://github.com/microsoft/TypeScript/pull/12253
+      const name = _name as keyof typeof standardPkg.peerDependencies
+      const ourRange = ourPkg.peerDependencies[name]
+      t.is(ourRange, standardRange)
+    })
+})
+
+test('Dependency @typescript-eslint/eslint-plugin', (t) => {
+  const peerDepPluginRange = ourPkg.peerDependencies['@typescript-eslint/eslint-plugin']
+  const devDepPluginRange = ourPkg.devDependencies['@typescript-eslint/eslint-plugin']
+  t.true(peerDepPluginRange.startsWith('>='))
```

Comment:
Could anyone please remind me why we're using this as range specifier in peerDependencies and not caret?

Reply:
`>=` also allows new major versions, not sure if we want that though 😄

Reply:
Thank you, @LinusU. No, I don't feel that we do. We can't support that.